static/app/views/alerts/list/rules/alertRulesList.tsx

```diff
@@ -183,94 +182,79 @@ function AlertRulesList() {
               onChangeFilter={handleChangeFilter}
               onChangeSearch={handleChangeSearch}
             />
-            <Teams provideUserTeams>
-              {({initiallyLoaded: loadedTeams, teams}) => (
```

Comment:
this is the main change in this file

recommend hiding whitespace changes

Reply:
Thanks for the recommendation!